stack_master.gemspec

```diff
@@ -20,8 +20,14 @@ Gem::Specification.new do |spec|
   spec.email         = ["steve@hodgkiss.me", "gstamp@gmail.com"]
   spec.summary       = %q{StackMaster is a sure-footed way of creating, updating and keeping track of Amazon (AWS) CloudFormation stacks.}
   spec.description   = %q{}
-  spec.homepage      = "https://github.com/envato/stack_master"
+  spec.homepage      = "https://opensource.envato.com/projects/stack_master.html"
```

Comment:
Given we can specify the source code URI below. Should this be the homepage?

https://opensource.envato.com/projects/stack_master.html

Reply:
I believe we're about to deprecate this domain, but it hasn't been migrated yet.